nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -53,6 +53,8 @@ def __init__(self, args):
         self.deployer = SimulatorDeployer()
         self.client_names = []
 
+        self.args.set = []
```

Comment:
Do we need this `self.args`?

Reply:
yes, need this to create the client. Share the same codes.